src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/runMatlabTests.m

```diff
@@ -32,35 +34,33 @@
 import('matlab.unittest.TestRunner');
 runner = TestRunner.withTextOutput;
 
-% Add the requested plugins
-resultsDir = fullfile(pwd, 'matlabTestArtifacts');
+
 
 % Produce JUnit report
-if produceJUnit
+if ~isempty(junitReportPath)
```

Comment:
The more I look at this, the more I think that it needs refactoring. Again, not something that needs to be done now, but something worth considering.

Reply:
Really we should remove this file entirely from resources and leverage the script generator. It should already be extensible, works across all CI platforms we support, and produces a nice simple script with non conditional code that the customer can read and leverage/reuse/modify/etc. Said another way, I think the refactoring has been done already, we just need to leverage it. I think delivering pipelines support is the priority here, but theres a follow up task this release to use the scriptgen infrastructure.